internal/worker/worker.go

```diff
@@ -211,3 +219,9 @@ func fetchBatch[T any](RPC common.RPC, blockNumbers []*big.Int, method string, a
 
 	return results
 }
+
+// Add a gauge to track the fetched block numbers in Prometheus
+var lastFetchedBlock = promauto.NewGauge(prometheus.GaugeOpts{
+	Name: "worker_last_fetched_block_from_rpc",
+	Help: "The last block number fetched by the worker from the RPC",
+})	
```

Comment:
Don't you want to create a package file for defining these metrics instead of defining these metrics everywhere?